extensions/EXT/EXT_fragment_shading_rate.txt

```diff
@@ -633,15 +633,16 @@
                                                                                                   each texel in a fragment shading
                                                                                                   rate attachment
     MAX_FRAGMENT_SHADING_RATE_ATTACHMENT_LAYERS_EXT             Z+    GetIntegerv  1              maximum number of layers in a
-                                                                                                  fragment shading rate attachment    13.X.3     
+                                                                                                  fragment shading rate attachment    13.X.3
+    FRAGMENT_SHADING_RATE_ATTACHMENT_WITH_-                     B    GetBooleanv  -               support for attachment shading      13.X.3
+    DEFAULT_FRAMEBUFFER_SUPPORTED_EXT                                                             rates on the default framebuffer
+[[If GL_EXT_fragment_shading_rate is supported]]

```

Comment:
Maybe also move the 
>[[If GL_EXT_fragment_shading_rate_attachment is supported]]

statement from line 609 to line 614 - so it groups all the attachment states?

Reply:
Nice. And the patch is updated accordingly.